### PR TITLE
Prepare for 0.1.0 release

### PR DESCRIPTION
**Pull request by sophokles73**
Several small fixes and improvements in preparation of the 0.1.0 release which is then to be published to crates.io.

This is for #15

### CODE REVIEW

File: src/utransport.rs [hunk 1]
```diff
@@ -38,103 +38,45 @@ pub trait LocalUriProvider: Send + Sync {
...
-    /// block further message receipt. For long-running operations consider passing off received
-    /// data to a different async function to handle it and returning.
-    ///
-    /// # Note for `UTransport` implementers
-    ///
-    /// Because `on_receive()` is async you may choose to either `.await` it in the current context
-    /// or spawn it onto a new task and await there to allow current context to immediately continue.
+    /// This function is expected to return almost immediately. If it does not, it could potentially
+    /// block processing of succeeding messages. Long-running operations for processing a message should
+    /// therefore be run on a separate thread.
```
Comment (AnotherDaniel) [inline src/utransport.rs:58]: '... thread' - or rather 'as a separate task'?

**Review by AnotherDaniel — APPROVED** [review]
LGTM, one minor editorial comment.

Excited!

File: src/utransport.rs (hunk 1 above)
Comment (sophokles73) [reply src/utransport.rs:58]: If you want to run blocking code, then you should probably use a dedicated (and separate) thread for running the code. Whether that code is scheduled as a task or run by means of message passing, is not relevant in this context FMPOV.

Comment (AnotherDaniel) [reply src/utransport.rs:58]: The thought was that 'task' represents a bigger option space for that might be done - essentially the same argument that you're making, but my word associations between task and thread seem to be the other way around :-)

**Pull request merged**